charts/redpanda/values.yaml

```diff
@@ -691,31 +691,39 @@ statefulset:
     runAsUser: 101
     fsGroupChangePolicy: OnRootMismatch
   sideCars:
+    # -- To create `Guaranteed` Pods for Redpanda brokers, provide both requests and limits for CPU and memory. For details, see
+    # https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed
+    # * Every container in the Pod must have a memory limit and a memory request.
+    # * For every container in the Pod, the memory limit must equal the memory request.
+    # * Every container in the Pod must have a CPU limit and a CPU request.
+    # * For every container in the Pod, the CPU limit must equal the CPU request.
+    #
+    # To maximize efficiency, use the `static` CPU manager policy by specifying an even integer for
+    # CPU resource requests and limits. This policy gives the Pods running Redpanda brokers
+    # access to exclusive CPUs on the node. For details, see
+    # https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/#static-policy
+    resources: {}
+    securityContext: {}
+    extraVolumeMounts: ""
+    image:
```

Comment:
This already exists in the controllers stanza. Do we want to duplicate it?

Reply:
The plan is to deprecate the entirety of the `controllers` stanza as this will _always_ get run due to the change in the health-checking behavior, meaning that the controller portions of the sidecar (i.e. pvc unbinding and decommissioning) are just implementation details of the actual "sidecar" itself. So I promoted the image to a higher stanza since it affects everything running in the sidecar now.